lighthouse-core/test/audits/final-screenshot-test.js

```diff
@@ -9,19 +9,46 @@ const assert = require('assert').strict;
 
 const FinalScreenshotAudit = require('../../audits/final-screenshot.js');
 const pwaTrace = require('../fixtures/traces/progressive-app-m60.json');
+const noScreenshotsTrace = {traceEvents: pwaTrace.traceEvents.filter(e => e.name !== 'Screenshot')};
 
 /* eslint-env jest */
 
 describe('Final screenshot', () => {
+  let context;
+
+  beforeEach(() => {
+    context = {computedCache: new Map()};
+  });
+
   it('should extract a final screenshot from a trace', async () => {
     const artifacts = Object.assign({
       traces: {defaultPass: pwaTrace},
+      GatherContext: {gatherMode: 'timespan'},
     });
-    const results = await FinalScreenshotAudit.audit(artifacts, {computedCache: new Map()});
+    const results = await FinalScreenshotAudit.audit(artifacts, context);
 
     assert.equal(results.score, 1);
     assert.equal(results.details.timing, 818);
     assert.equal(results.details.timestamp, 225414990064);
     assert.ok(results.details.data.startsWith('data:image/jpeg;base64,/9j/4AAQSkZJRgABA'));
   });
+
+  it('should returns not applicable for missing screenshots in timespan mode', async () => {
+    const artifacts = {
+      traces: {defaultPass: noScreenshotsTrace},
+      GatherContext: {gatherMode: 'timespan'},
+    };
+
+    const results = await FinalScreenshotAudit.audit(artifacts, context);
+    assert.equal(results.notApplicable, true);
```

Comment:
Can I lobby to remove `assert` from this test file? It was only 1 test.

Reply:
you can *always* lobby to remove `assert` :D